### ISSUE TITLE
FE | Profile | Default Send/Checkboxes & Editing as a Subtask | GA - Adjust checkbox events

### ISSUE THREAD

**Issue by adamwhitlock1** [department-of-veterans-affairs/va.gov-team#63727]
## Issue Description
This is a continuation of issue #61112

In looking at the GA events that are reported when using the `va-checkbox` component, the checkbox group isn't reported in the event, because we were wrapping the checkboxes in our own custom group to allow our required styling and api request based UI flow.

To get valuable GA event data we need to do one of 2 things:

- Use the `va-checkbox-group` component, and attempt to style and wrap the component in the needed code to fit into our needs on the Notification Settings page.
- Disable the GA events for the `va-checkbox` and fire our own event when the input value changes, so that we can pass in the required data for tracking.

## Front end tasks
- Attempt to use `va-checkbox-group` as it is the official way to get events firing correctly, and if that still does not fit our needs, then use a custom event.
- Verify that events are firing in staging, including the Checkbox name, value, and parent/group name

## Acceptance Criteria
- [x] GA events are seen in the GA dashboard for checkboxes
- [x] Unit tests pass
- [x] E2E tests cover current code and regression
- [x] End-to-end tests show 0 violations.

**Comment by adamwhitlock1** [comment]
I fired off a bunch of checkbox clicks on a staging user. Events look like this:

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdjB2IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4827ac9dd0ecb028e23b11d522d1f35938b9fc0a/Screenshot%202023-08-23%20at%2012.28.43%20PM.png)

So I can confirm that we are now sending the notification item name along with the other event data. I will follow up on this tomorrow through the GA dashboards to check if they are visible there as well.

**Comment by adamwhitlock1** [comment]
I can see the events firing now in staging GA through the realtime dashboards, but not in the main dashboard section. I will check tomorrow since I just updated the event naming and it will take a bit to get into the main section dashboards I guess.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaGROIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--a38f2339797eecb1e5fa2749f044524ef69b86e4/Screenshot%202023-08-29%20at%202.02.00%20PM.png)

**Comment by adamwhitlock1** [comment]
Newest GA dashboard from today:

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcHhUIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--3d23f5dcf6c5ba8ded15c1f3f89d5a7b9dae4245/Screenshot%202023-08-30%20at%208.57.42%20AM.png)

I'm not crazy about the false - false or true - false at the end of each event. I can try one more time to tweak that event to not have that kind of thing showing. 

Basically the second 'false' on each event is supposed to be whether the checkbox is required, and none of ours are required, so they are always going to be false and just muddy up the events and make it hard to read. 

Maybe I can just set it to a - or an empty space and see if I can get that to go away.

**Comment by mtcA6** [comment]
yea was going to ask about that but figured we didn't need to spend the time on the planning call. 

so the first false is the one we care about? Maybe instead of worrying about renaming the event maybe we group the event in a widget of "toggled on" or "toggled off"

**Comment by mtcA6** [comment]
@adamwhitlock1 can you share the link to that dashboard, i thought I had it but now I don't see it

**Comment by adamwhitlock1** [comment]
https://analytics.google.com/analytics/web/?authuser=3#/dashboard/DRqBrmiyTD6l8L75rei0fw/a50123418w177519031p176188361/

I realized that the link to these dashboards has this query parameter of `authuser=3` in it, and so if the link does not work, you may need to change that to `1` or something to get it to load for you.

This was the prod dashboard this morning:
![Screenshot 2023-09-07 at 8 52 51 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/8332986/80364d12-36ed-4d59-bbcd-2014e7e0d66d)

I'm guessing that is from doing QC work on prod briefly. I did put in a small PR to remove the `false` at the end of the event to hopefully make the events less noisy to read when there are several stacked up.
